app/views/account/videos/_video.html.slim

```diff
@@ -0,0 +1,2 @@
+.card-text.col-md-4.mb-2
```

Comment:
try make classes chain in order
bootstrap -> yours

also, it's not really good idea to strip part of grin into separate partial

Reply:
Actually, this is all bootstrap classes